counsel.el

```diff
@@ -1654,6 +1654,11 @@ Does not list the currently checked out one."
   :type 'boolean
   :group 'ivy)
 
+(defcustom counsel-preselect-current-file t
+  "When non-nil, preselects current file in list of candidates."
```

Comment:
Nitpick: "preselect" instead of "preselects", as this `defcustom` is not the one performing the preselection.

Reply:
Fixed below.